pkg/crc/cluster/cluster.go

```diff
@@ -153,6 +189,21 @@ Environment=NO_PROXY=.cluster.local,.svc,10.128.0.0/14,172.30.0.0/16,%s`
 	if err != nil {
 		return err
 	}
+
+	if proxy.ProxyCACert != "" {
+		logging.Debug("Adding proxy CA cert to instance")
+		return addProxyCACertToInstance(sshRunner, proxy)
+	}
+	return nil
+}
+
+func addProxyCACertToInstance(sshRunner *ssh.Runner, proxy *network.ProxyConfig) error {
+	if err := sshRunner.SetTextContentAsRoot("/etc/pki/ca-trust/source/anchors/openshift-config-user-ca-bundle.crt", proxy.ProxyCACert, 0600); err != nil {
```

Comment:
We do this manually because this should be done by the machine-config-operator, which we are not running?

Reply:
@cfergeau That's right